onedal/linear_model/incremental_linear_model.py

```diff
@@ -72,24 +72,28 @@ def partial_fit(self, X, y, queue=None):
         self : object
             Returns the instance itself.
         """
+        if not hasattr(self, "_params"):
+            self._params = self._get_onedal_params(X.dtype)
+
+        if _get_config().get("use_raw_input") is False:
+            X, y = _check_X_y(
+                X,
+                y,
+                dtype=[np.float64, np.float32],
+                accept_2d_y=True,
+                force_all_finite=False,
+            )
+            y = np.asarray(y, dtype=X.dtype)
+
         module = self._get_backend("linear_model", "regression")
 
-        self._queue = queue
         policy = self._get_policy(queue, X)
-
-        X, y = _check_X_y(
-            X, y, dtype=[np.float64, np.float32], accept_2d_y=True, force_all_finite=False
-        )
-        y = np.asarray(y, dtype=X.dtype)
+        queue = self._queue = getattr(policy, "_queue", None)
```

Comment:
a little confused on this line - line 90 has policy set based on queue, then line 91 sets queue based on policy?